src/main/scala/com/ibm/couchdb/api/builders/GetManyDocumentsQueryBuilder.scala

```diff
@@ -160,13 +230,68 @@ case class GetManyDocumentsQueryBuilder(
   }
 
   private def queryWithoutIds[Q: R](ps: Map[String, String]): Task[Q] = {
-    postQuery[Q](client, db, s"/$db/_all_docs", ps)
+    query[Q](client, s"/$db/_all_docs", ps)
   }
 
   private def queryByIds[Q: R](ids: Seq[String], ps: Map[String, String]): Task[Q] = {
     if (ids.isEmpty)
       Res.Error("not_found", "No IDs specified").toTask
     else
-      queryByIds[String, Q](client, db, s"/$db/_all_docs", ids, ps)
+      queryByIds[String, Q](client, s"/$db/_all_docs", ids, ps)
+  }
+}
+
+object GetManyDocumentsQueryBuilder {
+
+  type MDBuilder[ID <: DocsInResult, AM <: MissingIdsInQuery, BT <: DocType] =
+  GetManyDocumentsQueryBuilder[ID, AM, BT]
+
+  case class Builder[T: R, ID <: DocsInResult, AM <: MissingIdsInQuery]
+  (builder: MDBuilder[ID, AM, AnyDocType]) {
+    def build: QueryBasic[T] =
+      new QueryBasic[T](builder.client, builder.db, builder.params, builder.ids)
   }
+
+  case class ByTypeBuilder[K: R, V: R, D: R](
+      builder: MDBuilder[IncludeDocs[D], MissingNotAllowed, ForDocType[K, V, D]])
+      (implicit tag: ClassTag[D], kw: W[K]) {
+    def build: QueryByType[K, V, D] =
+      builder.view match {
+        case Some(view) => new QueryByType[K, V, D](
```

Comment:
this should not happen :-) 
we can add a case for `None` that uses the defined temporary view, but this does not sound right to me since it is covering a more sinister flaw in the builder logic .. best to fail fast if such a thing occurs